components/ContentFlow.brs

```diff
@@ -269,31 +269,21 @@ function unpackStreamInformation() as boolean
 end function
 
 '-------------------------------------------------------------------------------------------------------------
-' Determines the URL string used to request a VAST config. The full URL is created by appending the following
-' URL parameters to the provided baseUrl:
-'   * network_user_id = userId, provided
-'   * stream_position = either "preroll" or "midroll" depending on ad slot
-'   * env[] = appends "brightscript" and "layoutJSON" elements to support Roku
+' Determines the URL string used to request a VAST config. This is typically passed through from the 
+' ad parameters stored in the VAST payload or companion ad.
 '
 ' Params:
 '   * baseUrl as string - URL of VAST config URL parameters will be appended to
-'   * userId as string - identifier used as value for network_user_id URL parameter
 '
 ' Return:
 '   full URL to use for VAST config GET request
 '-------------------------------------------------------------------------------------------------------------
-function determineVastConfigUrl(baseUrl as string, userId as string) as string
+function determineVastConfigUrl(baseUrl as string) as string
     ? "TRUE[X] >>> ContentFlow::determineVastConfigUrl(baseUrl=";baseUrl;", userId=";userId;")"
 
     ' prepend HTTP protocol if it's absent
     if Left(baseUrl, 4) <> "http" then baseUrl = "https://" + baseUrl
 
-    ' append URL parameters; network_user_id, stream_position, env[]
-    baseUrl = baseUrl + "&network_user_id=" + getClientAdvertisingId()
-    baseUrl = baseUrl + "&stream_position=" + getCurrentAdBreakSlotType()
-    baseUrl = baseUrl + "&env%5B%5D=brightscript"
-    baseUrl = baseUrl + "&env%5B%5D=layoutJSON"
```

Comment:
It was my understanding that the client TAR was expected to append these `env[]` values -- is that not the case?

Reply:
Yes, this is why we're removing this code which previously appended them from the reference app. This is previously reviewed changes which were picked up by this PR since we're holding on a couple of updates to the ref app pending TAR v1.1 release.